src/views/Scan.vue

```diff
@@ -535,6 +600,10 @@
 					this.resetFilters();
 					break;
 				}
+				case 'u':
```

Comment:
I would prefer `f`, which is already used for filters, which in retrospective may not have been the best decision. @ipf, what do you think, rebind, stay with `u` or use another key (combination)?

Reply:
I'd prefer this as non-breaking. So, if *u* is documented as key binding for fullscreen-toggling, that's fine for me.